django/core/mail/message.py

```diff
@@ -190,6 +191,10 @@ def __setitem__(self, name, val):
         MIMEMultipart.__setitem__(self, name, val)
 
 
+Alternative = namedtuple("Alternative", ["content", "mime_type"])
```

Comment:
```suggestion
Alternative = namedtuple("Alternative", ["content", "mimetype"])
```
This is currently documented as `mimetype`, test update is required

Reply:
Damn, I updated one and missed the other :sweat_smile: